composer.json

```diff
@@ -18,13 +18,13 @@
         }
     ],
     "require": {
-        "php": "^7.2|^8.0",
-        "illuminate/contracts": "^6.0|^7.0|^8.0",
-        "illuminate/http": "^6.0|^7.0|^8.0",
-        "illuminate/support": "^6.0|^7.0|^8.0"
+        "php": "^8.0",
```

Comment:
Minimum PHP version should be `^8.0.2`

Reply:
yup, scrutinizer also doesn't like 8.0 :)